src/components/dynamic-label/dynamic-label.scss

```diff
@@ -25,8 +23,9 @@ limel-icon {
 }
 
 span {
-    @include mixins.truncate-text;
     flex-grow: 1;
     font-size: 0.8125rem; // `13px`, Like Checkbox & Switch
+    letter-spacing: 0.02em;
```

Comment:
@TommyLindh2 @LucyChyzhova I'm really glad that you guys also noticed these details. 🤗 

I'm aware of them, which is why I wrote "minimize" "[checkbox: minimize the layout shifts in readonly mode](https://github.com/Lundalogik/lime-elements/pull/3277/commits/f337ef1ccf6a94cbef73ade9133ce0a893bb7c6d)" for the commit message. The thing is that we are gonna remove all MDC stuff from Lime Elements. Switch and Checkbox are two of those components that should also be de-MDCfied, as they come with bugs and limitations. Many styles, specially typographic styles including the letter spacing is coming from the MDC, which cause different renderings.

I didn't want to change these styles, to make the future work a bit simpler. But @LucyChyzhova if you want to change the typographic styles such as the letter spacing, please do it in the Checkbox and Switch instead of inside Dynamic Label. 🙏

Reply:
> I didn't want to change these styles, to make the future work a bit simpler. But @LucyChyzhova if you want to change the typographic styles such as the letter spacing, please do it in the Checkbox and Switch instead of inside Dynamic Label. 

So you mean we should not apply it to span, but to the checkbox and switch labels instead?

Reply:
yes exactly. thanks :)